### ISSUE TITLE
Assertion failed UnityEditor.RetainedMode:UpdateSchedulers 

### ISSUE THREAD

**Issue by KAMIKAZEUA** [googleads/googleads-mobile-unity#1786]
### [REQUIRED] Step 1: Describe your environment

  * Unity version: 2020.3.19f1
  * Google Mobile Ads Unity plugin version: 6.1.1
  * Platform: iOS, Unity Editor
  * Platform OS version: Big Sur 11.6 (20G165)
  * Any specific devices issue occurs on: editor
  * Mediation ad networks used, and their versions: AdColony 2.3.0, Applovin 5.1.1, Facebook Audience Network
3.4.0, IronSource 2.1.1, Tapjoy 2.7.0, Unity Ads 2.7.2, Vungle 3.6.0  

#### Steps to reproduce:

We integrated 6.1.0 with all that mediation networks and submitted production build. All ok, all works as expected. 

Then we tried to update to 6.1.1, update was like okay, but then we tried to build Xcode project - got error:
<img width="267" alt="Screenshot 2" src="https://user-images.githubusercontent.com/3391456/135859462-f4c5ede6-31bb-4d0c-950b-e4e30c3e5205.png">

Clicked - Open Settings gave us iOS/android ID's now **empty**.
Well, we filled them again, build project, build ok. Then **close Unity** and check for file modifications on git: 

<img width="703" alt="Screenshot" src="https://user-images.githubusercontent.com/3391456/135857127-8351baa5-5dcb-4013-9662-ed98ce793511.png">

As you can see, after update of Admob to 6.1.1 it's wiped out project ID's, even after we entered them manually and saved project.

So, again we just tried to select file in editor **GoogleMobileAdsSettings.asset** and got 999+ errors: 
Assertion failed
UnityEditor.RetainedMode:UpdateSchedulers () (at /Users/bokken/buildslave/unity/build/External/MirroredPackageSources/com.unity.ui/Editor/RetainedMode.cs:50)

<img width="717" alt="Screenshot 1" src="https://user-images.githubusercontent.com/3391456/135857363-145121e2-ca77-4e46-a4e3-c578af691baa.png">

So, in short steps was:
- update admob - it's wiped ID's, ok, we re-enter ID's again in GoogleMobileAdsSettings.asset file
- close/open project, to check if ID's actually saved(also we tried in menu Unity: File -> Save Project)
- select **GoogleMobileAdsSettings.asset** cause 999+ errors,

Unable to edit this file now, always errors if we try to select it or if we try to build Xcode project.

**Comment by Bagdaja** [comment]
Have you tried downgrading the admob version? I've tried and when selecting GoogleMobileAdsSettings.asset it didn't show any errors, but when trying to play in the editor unity will simply crash

**Comment by KAMIKAZEUA** [comment]
We just reverted commit in our project with latest update try to 6.1.1
We don’t see any reason to use broken version in our production releases.

**Comment by sammorris-fss** [comment]
We're getting the same thing here. Unity 2020.3.19 and plugin version 6.1.1. Makes the SDK entirely unusable.

**Comment by 96698041** [comment]
Same to me!
Editor: 2020.3.19f1 - Windows,
Platform: Android,
Admob plugin version: 6.1.1
Error: It's wiped out project ID's, even after we entered them manually and saved project and selecting file in editor GoogleMobileAdsSettings.asset... 999+ errors Assertion failed: UnityEditor.RetainedMode:UpdateSchedulers ()

**Comment by KAMIKAZEUA** [comment]
@NVentimiglia one more highest priority.

**Comment by yerfdogj** [comment]
Related with fixes:
https://github.com/googleads/googleads-mobile-unity/issues/1796
https://github.com/googleads/googleads-mobile-unity/issues/1801

**Comment by NVentimiglia** [comment]
Pushed a release 6.1.2 which should solve the issue. Please let me know if you run into any further issues.

https://github.com/googleads/googleads-mobile-unity/releases/tag/v6.1.2

**Comment by afsanchez001** [comment]
This happened to me after upgrading to 6.1.1 and generated 999+ messages. After closing Unity and restarting it, the error disappeared.

**Comment by xtrecoolx** [comment]
The issue still exists for Android. Admob settings are not saved.
